libraries/src/HTML/HTMLHelper.php

```diff
@@ -748,6 +748,46 @@ public static function stylesheet($file, $options = array(), $attribs = array())
 		}
 	}
 
+	/**
+	 * Write a `<span>` element for an icon
+	 *
+	 * @param   string        $icon        The funcitonal name for an icon.
+	 * @param   array         $attribs     Attributes to be added to the wrapping element
+	 * @param   string        $iconPrefix  String prepend to icon className
+	 * @param   string        $iconSuffix  String append to icon className
+	 * @param   string        $elem        String that indicates the wrapping element. Default `span`
+	 * @param   string        $children    String append to icon className
+	 *
+	 *
+	 * @return  string
+	 *
+	 * @since   4.0
+	 */
+	public static function icon($icon = 'default', $attribs = [], $iconPrefix = 'icon-', $iconSuffix = null, $html = true, $elem = 'span', $children = null)
+	{
+		$icon = $iconPrefix . $icon;
+
+		if (isset($iconSuffix))
+		{
+			$icon = $icon . $iconSuffix;
+		}
+
+		if (!isset($attribs['aria-hidden']))
+		{
+			$attribs['aria-hidden'] = 'true';
+		}
+
+		$attribs['class'] =  $iconPrefix . $icon;
```

Comment:
```suggestion
		$attribs['class'] = $iconPrefix . $icon;
```
Fix PHP code style errors reported by Drone here https://ci.joomla.org/joomla/joomla-cms/35832/1/8